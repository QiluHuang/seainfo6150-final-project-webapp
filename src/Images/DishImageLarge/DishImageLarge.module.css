.img {
    width: 300px;
    height: 250px;
    margin-right: 10px;
    object-fit: cover;
    /* background: red; */
    border-radius: 200px / 20px;
    /* border: 10px solid #ca0000; */
    box-shadow: 0px 10px 10px gray;
}

@media screen and (min-width: 525px) {
    .image {
      width: 65%; 
    }
  }
  
  @media screen and (min-width: 800px) {
    .image {
      width: 50%;
    }
  }
  
  @media screen and (min-width: 1150px) {
    .image {
        width: 45%;
    }
  }
  