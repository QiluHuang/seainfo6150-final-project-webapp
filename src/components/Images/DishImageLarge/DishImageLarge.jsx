import React from "react";
import PropTypes from "prop-types";
import styles from "./DishImageLarge.module.css";

const DishImage = (props) => {
    return (
        <img className={styles.img} src={props.url} alt={props.title} />
    )
}

DishImage.prototype = {
    url: PropTypes.string.isRequired,
    title: PropTypes.string.isRequired,
};
export default DishImage;