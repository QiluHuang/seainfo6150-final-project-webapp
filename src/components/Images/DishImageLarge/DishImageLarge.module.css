.img {
    width: 30vw;
    height: auto;
    border-radius: 20px;
}


  