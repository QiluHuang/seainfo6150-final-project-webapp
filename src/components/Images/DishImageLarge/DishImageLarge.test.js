import React from "react";
import { render } from "@testing-library/react";
import DishImageLarge from "./DishImageLarge.jsx";

const imageURL = "https://d.newsweek.com/en/full/1552244/variety-chinese-food.jpg";
const title = "soup";

describe("DishImageLarge tests", () => {
    it("renders the DishImageLarge component correctly", () => {
    const { container } = render(<DishImageLarge url={imageURL} title={title} />);
    expect(container).toMatchSnapshot();
    });
});