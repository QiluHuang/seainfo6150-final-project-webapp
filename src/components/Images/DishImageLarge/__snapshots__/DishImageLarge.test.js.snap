// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DishImageLarge tests renders the DishImageLarge component correctly 1`] = `
<div>
  <img
    alt="soup"
    class="img"
    src="https://d.newsweek.com/en/full/1552244/variety-chinese-food.jpg"
  />
</div>
`;
